### ISSUE TITLE
broken recipes for sticks/planks

### ISSUE THREAD

**Issue by Shivaxi** [alcatrazEscapee/no-tree-punching#85]
It seems recipes got completely broken for using the saw in a crafting table with wood and planks to make planks and sticks.

It also overrides my own recipe that I made for birch wood giving paper using crafttweaker, doesn't work at all anymore which is strange.

**Comment by Shivaxi** [comment]
oh and, i dont know if its intended but, the fix for double plant flowers not dropping grass fiber works, but it also disables it for double plant grass.  im not sure if you can differentiate between the two however.  not a big issue really

**Comment by Shivaxi** [comment]
okay also noting that right clicking birch wood with an axe gives paper as well instead of birch planks haha, because of my recipe i guess in crafttweaker where 1 birch makes 3 planks

this is the recipe i have btw in crafttweaker:

recipes.addShapeless("lolarecipe21",<minecraft:paper>*3,[<minecraft:log:2>]);

**Comment by alcatrazEscapee** [comment]
NTP will search through all registered recipes, and try and use them to create a plank from wood. It then will remove this recipe, and replace it with a saw recipe, along with noting the wood -> plank mapping for the purposes of wood chopping. So yes, if you add a recipe that looks similar to a log -> plank recipe, it will get caught as a wood -> plank recipe.

If you have other issues, please open separate issue reports for each individual issue. Include version + exactly what's not working please.

**Comment by Shivaxi** [comment]
in 2.0.13 this was not an issue though, and I could have my birch log -> paper recipe just fine?

but as it stands with NTP 2.0.18, none of the saw recipes are working, whether replaceLogRecipes is true or false

**Comment by Shivaxi** [comment]
I'm also not sure you understand.  simply by having this now:

recipes.addShapeless("lolarecipe21",minecraft:paper*3,[minecraft:log:2]);

in crafttweaker, is making it so that right clicking a birch log with an axe is dropping paper instead of planks.  furthermore its effecting the saw recipe in 2.0.17, saw on top of birch wood now giving paper.  it's having a different effect in 2.0.17 than 2.0.18, and i can't seem to fix either one manually in crafttweaker without SOMETHING being broken.  Was hoping to release RLCraft v2.7 tonight but we discovered this in a public test and had to delay it further.

**Comment by alcatrazEscapee** [comment]
Yes, I understand. The fix for #75 involved beefing up the recipe detection, so it is apparently now catching craft tweaker recipes as well. I realize this is not ideal especially for pack makers, so I'm considering adding a) an option to disable the checks that are running after CT applies it's recipe changes. and b) a manual way to specify wood -> log recipes

**Comment by Shivaxi** [comment]
alternatively, what about instead of beefing up the recipe detection, can there be some sort of whitelist where we can add tools/wood to a list?  So like if I want to be able to chop Fir Wood from Traverse, I just add traverse:fir_log into a whitelist in NTP.  Something like that.  I feel like that could be a lot easier for pack makers to add custom woods and stuff like this.

**Comment by alcatrazEscapee** [comment]
By default I'd like this to work without any customization - you should just be able to add NTP to a pack without having to manually change all the logs for all mods. So I can disable the recipe detection, but it would be enabled by default.
As for tool detection, ore dictionary values should work (although I accidentally removed them in the last update, which apparently broke saw recipes. I'll fix that asap.). Although it should be very good now at detecting modded axes of all kinds.
I can add a craft tweaker hook for chopping.

**Comment by Shivaxi** [comment]
ah gotcha.  I released RLCraft v2.7 with NTP 2.0.18 for now, I couldn't wait any longer as there was a serious crash fix I needed to push out.  For now I manually added back every single saw recipe by hand using crafttweaker haha.  Just gotta adapt sometimes =P  Also changed my paper recipe to take 2 birch wood instead of 1 to avoid the conflict for now.  Will certainly update NTP again for RLCraft v2.8!

Appreciate the speedy updates btw, I know things have been crazy with nearly a million players downloading your mod and others haha.  Been working with a lot of the devs now to get bugs sorted =P just been nuts.  Really glad you're still active on this!
